packages/sdk/src/react/hooks/utils.ts

```
@@ -0,0 +1,64 @@
+/**
+ * Performs a deep comparison between two values to determine if they are equivalent.
+ * @param a First value to compare
+ * @param b Second value to compare
+ * @returns True if the values are equivalent, false otherwise
+ */
+export function deepEqual(a: any, b: any): boolean {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)